js/privatebin.js

```diff
@@ -5328,6 +5571,23 @@ jQuery.PrivateBin = (function($, RawDeflate) {
                 SAFE_FOR_JQUERY: true
             });
 
+            // Add a hook to make all links open a new window
```

Comment:
This implements (pretty elegantly) the feature discussed in https://github.com/PrivateBin/PrivateBin/issues/61

Is this why you applied DOMpurify in url2links above? If so, we would just need to find a way to add magnet link support to it.

Reply:
Yes this is part of the reason, at the meantime I think all direct `.html()` calls should be wrapped with `DOMpurify.sanitize()`.